### PR TITLE
Added GPS FeatherWing

### PR DESCRIPTION
**Pull request by makermelissa**
Ok, here's the GPS FeatherWing code. It handles the initialization automatically, makes some properties shorter names and has some speed calculations such as providing the speed in MPH and KPH.

### CODE REVIEW

**Comment by makermelissa** [comment]
Also, I have tested this on the Feather M4 Express and the Feather nRF52840 Express.

File: adafruit_featherwing/gps_featherwing.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+        self._uart = busio.UART(shared.TX, shared.RX, baudrate=baudrate, timeout=timeout)
+        self._gps = adafruit_gps.GPS(self._uart, debug=False)
+        # Turn on the basic GGA and RMC info
+        self._gps.send_command(b'PMTK314,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0')
+        self._gps.send_command(b'PMTK220,{}'.format(update_period))
+
+    def update(self):
+        """
+        Make sure to call gps.update() every loop iteration and at least twice
```
Comment (kattni) [inline adafruit_featherwing/gps_featherwing.py]: If you wan it to show up as code in the docs, you can put double backticks on either side (``). Single backticks will cause sphinx to attempt to do something with it, double tells sphinx that you're trying to display something in fixed-width font.

**Review by kattni — CHANGES_REQUESTED** [review]
Looks great! One minor cosmetic suggestion.

File: adafruit_featherwing/gps_featherwing.py (hunk 1 above)
Comment (kattni) [reply adafruit_featherwing/gps_featherwing.py]: I'm referring to `gps.update()`.

Comment (makermelissa) [reply adafruit_featherwing/gps_featherwing.py]: Thanks! I didn't know that.

**Review by kattni — APPROVED** [review]
Perfect! Thanks!

**Pull request merged**